package.json

```diff
@@ -32,5 +32,13 @@
     "passport-discord": "^0.1.4",
     "passport-local": "^1.0.0",
     "validator": "^13.6.0"
-  }
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/InstincDev/Together.git"
```

Comment:
What is the functionality of this?

Reply:
Not sure. Probably got added while I was trying to get the app running.  🙃  You can delete or I will in the morning.

Reply:
removed it